config/locales/de.yml

```diff
@@ -531,6 +531,7 @@ de:
     submit:
       failure: Beim Übermitteln Ihrer Punktzahl ist ein Fehler aufgetreten. Bitte versuchen Sie es später erneut.
       too_late: Ihre Abgabe wurde erfolgreich gespeichert, ging jedoch nach der Abgabefrist ein.
+      full_score: Herzlichen Glückwunsch! Sie haben die maximale Punktzahl für diese Aufgabe erreicht. Klicken Sie <a href="%{url}" rel="nofollow">hier</a>, um die Aufgabe abzuschließen.
```

Comment:
Please add the `class="alert-link"` to the link, as this will be required with the newest style changes.